sonar-plugin-api/src/test/java/org/sonar/api/server/rule/RulesDefinitionTest.java

```diff
@@ -257,13 +260,12 @@ public void cant_set_blank_repository_name() {
   }
 
   @Test
-  public void warning_if_duplicated_rule_keys() {
+  public void fail_if_duplicated_rule_keys_in_the_same_repository() {
+    expectedException.expect(IllegalArgumentException.class);
```

Comment:
@julienlancelot and I find it a better practice to declare the expectedException right before the statement supposed to throw it
It avoids catching an exception raised by another statement and not have a test failure when we should and it's also better for readability of the UT (it shows where the exception is expected to be raised)